torchbenchmark/util/model.py

```diff
@@ -391,3 +392,7 @@ def enable_fx_int8(self, quant_engine:str='x86'):
         except Exception as e:
             print(e)
             raise RuntimeError(f"{self.name} doesn't support `fx_int8` yet!")
+
+    def enable_bf16(self):
+        self.model = self.model.to(torch.bfloat16)
+        self.example_inputs = (self.example_inputs[0].to(torch.bfloat16), )
```

Comment:
The conversion of `self.example_inputs` seems not general enough, because there are different type for it. We may use the similar method in `enable_channels_last`

Reply:
Done.